### PR TITLE
✨ feat(hotkeys): display platform specific key

### LINKED ISSUE #349: [Bug] 保存为话题快捷键在 Mac 上不生效
### 💻 系统环境 | Operating System

macOS

### 🌐 浏览器 | Browser

Chrome

### 🐛 问题描述 | Bug Description

保存为话题快捷键在 Mac 上不生效

image

### 🚦 期望结果 | Expected Behavior

_No response_

### 📷 复现步骤 | Recurrence Steps

_No response_

### 📝 补充信息 | Additional Information

_No response_

### PR DESCRIPTION
**Pull request by Asuka109**
#### 💻 变更类型 | Change Type

- [x] ✨ feat
- [ ] 🐛 fix
- [ ] ♻️ refactor
- [ ] 💄 style
- [ ] 🔨 chore
- [ ] 📝 docs

#### 🔀 变更说明 | Description of Change

Display platform specific key.

### CODE REVIEW

File: src/utils/platform.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import UAParser from 'ua-parser-js';
+
+export const getPlatform = () => {
+  const ua = navigator.userAgent;
```
Comment (arvinxx) [inline src/utils/platform.ts]: this seems break on server mode, because navigator is undefined on node.

Comment (arvinxx) [reply src/utils/platform.ts]: try it, it seems work correctly...

File: src/components/HotKeys/index.tsx [hunk 2]
```diff
@@ -26,14 +28,21 @@ const useStyles = createStyles(
   `,
 );
 
+const HOTKEY_MAPPINGS: Partial<Record<string, string>> = {
+  alt: isApplePlatform() ? 'option' : 'alt',
```
Comment (arvinxx) [inline src/components/HotKeys/index.tsx]: please use ⌥ on Mac

**Review by arvinxx — CHANGES_REQUESTED** [review]

File: src/utils/platform.ts (hunk 1 above)
Comment (Asuka109) [reply src/utils/platform.ts]: I will resolve it tomorrow.

File: src/components/HotKeys/index.tsx (hunk 2 above)
Comment (canisminor1990) [reply src/components/HotKeys/index.tsx]: > please use ⌥ on Mac

backspace could also use ⌫

Comment (Asuka109) [reply src/components/HotKeys/index.tsx]: done~

**Pull request merged**